include/GPM/Shape3D/AABB.hpp

```diff
@@ -0,0 +1,86 @@
+﻿/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ * found in the top-level directory of this distribution.
+ */
+
+#pragma once
+
+#include "../Tools.hpp"
+#include "Volume.hpp"
+#include "../Vector3.hpp"
+
+namespace GPM
+{
+class AABB : public Volume
+{
+protected:
+    Vec3  m_center;
+    float m_iI, m_iJ, m_iK;
```

Comment:
All of the data members have getters and setters. I suggest setting them `public`.
I would also suggest changing
```C++
float m_iI, m_iJ, m_iK;
```
to 
```C++
Vec3 bounds;
```